src/locutus/__init__.py

```diff
@@ -13,19 +13,59 @@ def strip_none(value):
     return value
 
 
-def fix_varname(varname):
-    return varname.strip().replace(" ", "_")
-
-
-def clean_varname(name):
-    return (
-        name.lower()
-        .replace(" ", "_")
-        .replace("(", "")
-        .replace(")", "")
-        .replace("'", "")
-        .replace('"', "")
-    )
+FTD_PLACEHOLDERS = {
+    "<FTD-DOT>": ".",
+    "<FTD-DOT-DOT>": ".."
+    }
+
+def normalize_ftd_placeholders(code):
+    """
+    Replaces special FTD placeholders if the entire code matches them.
+    
+    Args:
+        code(str): The input code.
+
+    Returns:
+        str: The normalized code.
+
+    """
+    if code in FTD_PLACEHOLDERS:
+        return FTD_PLACEHOLDERS[code]
+    else:

```

Comment:
I would return code for that else, so that you could just always process code without worrying ahead of time if it will be changed. (ie limit the number of times you have to test code is in placeholders. )